build/gen_stub.php

```diff
@@ -761,12 +767,22 @@ function(FuncInfo $funcInfo) use(&$generatedFuncInfos) {
 
     if ($fileInfo->generateFunctionEntries) {
         $code .= "\n\n";
-        $code .= generateCodeWithConditions($fileInfo->funcInfos, "", function(FuncInfo $funcInfo) {
+        $code .= generateCodeWithConditions($funcInfos, "", function(FuncInfo $funcInfo) use ($funcInfos) {
+            $result = "";
+
             if ($funcInfo->alias) {
-                return null;
+                foreach ($funcInfos as $info) {
```

Comment:
This solution doesn't have the best time complexity, but I supposed it wasn't a matter in this case.

Reply:
I'd prefer to do something along the lines of...

```php
$name = $funcInfo->alias ?: $funcInfo->name;
if (isset($generatedDeclarations[$name])) {
    return nullptr;
}

$generatedDeclarations[$name] = true;
return "ZEND_FUNCTION($name);\n";
```

Reply:
Your version is admittedly more elegant :) + I've just realized that my code generated a declare for the alias functions in vain.